### PR TITLE
Add Warsaw doc

### LINKED ISSUE #1878: Add Warsaw
[Warsaw.docx](https://github.com/slurpcode/slurp/files/9689683/Warsaw.docx)

https://slurp.readthedocs.io/en/latest/index.html

Author is Idais

### PR DESCRIPTION
**Pull request by Kevan-Y**
Fixes #1878 

## Description
- Add Warsaw doc

### CODE REVIEW

**Review by jbampton — CHANGES_REQUESTED** [review]
Please use `-` in filenames

**Comment by Kevan-Y** [comment]
> Please use `-` in filenames

Oupsy right, I got used to use `_`

File: doc/source/warsaw.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+![Warsaw](_static/images/warsaw/warsaw.png)
+
+We have made a list of the essential places you should visit if you go to Warsaw.
+
+## Old City
+
+One of the best activities you can do in Warsaw is to walk the streets of the Old Town, which is the oldest neighborhood and was completely in ruins during the Nazi occupation, but it was rebuilt so similar to the original that it was declared a World Heritage Site by the Unesco in 1980.
+In the heart of the city is located in the beautiful Old Town square where tourists and locals converge, in the surroundings you can see its cobbled streets with houses of medieval architecture and their colorful facades. You can have a drink in the restaurants with terraces that surround the square and see the street artists or painters that are usually in this area.
```
Comment (jbampton) [inline doc/source/warsaw.md:12]:
```suggestion

In the heart of the city is located in the beautiful Old Town square where tourists and locals converge, in the surroundings you can see its cobbled streets with houses of medieval architecture and their colorful facades. You can have a drink in the restaurants with terraces that surround the square and see the street artists or painters that are usually in this area.
```

File: doc/source/warsaw.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+## New City
+
+The New City has numerous points of interest, founded at the end of the 14th century, it developed around the New City square and is separated from the Old City by a fortress that surrounded the city in medieval times. In the square is the church of San Casimir characterized by its white facade and there is a statue of Marie Curie, born in this city and the first woman to receive two Nobel Prizes (physics and chemistry). If you are interested in knowing more about his life and work, you can visit his old home located on Freta Street, currently it is one of the most beautiful House-Museums to see in the city.
+The recommendation is to observe the most beautiful buildings that are on the banks of the Vistula River and then visit the most important churches such as San Jacek, the Holy Spirit and the Visitation of the Virgin Mary.
```
Comment (jbampton) [inline doc/source/warsaw.md:31]:
```suggestion

The recommendation is to observe the most beautiful buildings that are on the banks of the Vistula River and then visit the most important churches such as San Jacek, the Holy Spirit and the Visitation of the Virgin Mary.
```

**Review by jbampton — COMMENTED** [review]
Do you shrink the images ?

https://slurp--1908.org.readthedocs.build/en/1908/zurich.html

**Comment by Kevan-Y** [comment]
> Do you shrink the images ?
> 
> https://slurp--1908.org.readthedocs.build/en/1908/zurich.html

Nope, I saved it as it is. Did not touch the size of the image

**Comment by jbampton** [comment]
When I save the images they are larger.

Looks like your pr is different

**Comment by Kevan-Y** [comment]
@jbampton, Does my old build get replace by new one after a new commit?

If so, I can't seems to get the correct size. I'm not sure if it's OS issue or something else.

Steps of my file saving:
1. Open docx file
2. Click on the image
3. Save as Picture 
4. Rename and put in the correct folder

**Comment by jbampton** [comment]
Yes as long as you commit the work it does update.

But I grab the images from the docs differently.

I open the doc in `LibreOffice` and then click on an image and select copy.

Then I open `preview` image editor and select `new from clipboard`

Free and open source:  https://www.libreoffice.org/

**Comment by Kevan-Y** [comment]
Updated with the correct size of image. I do not know why microsoft office produce an incorrect image size. Good to know LibreOffice works. Thanks

**Review by jbampton — APPROVED** [review]

**Pull request merged**